src/Analyzers/CSharp/Analyzers/NewLines/ArrowExpressionClausePlacement/ArrowExpressionClausePlacementDiagnosticAnalyzer.cs

```diff
@@ -48,6 +51,9 @@ private void Recurse(SyntaxTreeAnalysisContext context, ReportDiagnostic severit
 
             foreach (var child in node.ChildNodesAndTokens())
             {
+                if (context.FilterSpan.HasValue && !child.Span.IntersectsWith(context.FilterSpan.GetValueOrDefault()))
```

Comment:
```suggestion
                if (context.FilterSpan.HasValue && !child.Span.IntersectsWith(context.FilterSpan.Value))
```

(just change them all ;))

Reply:
Added extension methods.